packages/components/src/internal/models.ts

```diff
@@ -1,20 +1,10 @@
-import { List, Map } from 'immutable';
 import { SchemaQuery } from '../public/SchemaQuery';
 
 export function createGridModelId(gridId: string, schemaQuery: SchemaQuery, keyValue?: any): string {
     const parts = [gridId, schemaQuery.getKey()];
-
```

Comment:
Note, I removed this since `schemaQuery.getKey()` by default includes an encoded view name if the `schemaQuery` has a view name specified.